extensions/raycast-ikea-idasen/package.json

```
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "raycast-ikea-idasen",
```

```suggestion
  "name": "ikea-idasen",
```
Rename this line and the extension folder